pkg/api/copy_test.go

```diff
@@ -32,24 +32,24 @@ import (
 
 func TestDeepCopyApiObjects(t *testing.T) {
 	for i := 0; i < *fuzzIters; i++ {
-		for _, gv := range []unversioned.GroupVersion{testapi.Default.InternalGroupVersion(), *testapi.Default.GroupVersion()} {
-			f := apitesting.FuzzerFor(t, gv.String(), rand.NewSource(rand.Int63()))
-			for kind := range api.Scheme.KnownTypes(gv) {
-				doDeepCopyTest(t, gv.String(), kind, f)
+		for _, version := range []unversioned.GroupVersion{testapi.Default.InternalGroupVersion(), *testapi.Default.GroupVersion()} {
+			f := apitesting.FuzzerFor(t, version.String(), rand.NewSource(rand.Int63()))
```

Comment:
> * `GroupVersion` is more like a `Package` if I were to take Java naming conventions.  This also make sense to me since if we start to support third-party API groups, I would want to add the other fields to a `Package` like vendor for example.

I think somewhere smarterclayton, lavalamp, and bgrant0607 has agreed vendor should be part of the GroupVersion.Group, and use "." to delimiter vendor and group.